### ISSUE TITLE
Theme update - sidebar disappears

### ISSUE THREAD

**Issue by CAMorgan** [mgsisk/inkblot#48]
Hi there! My site at http://tdugn.com has been updated to the latest Inkblot, but when I updated my sidebar completely disappeared and now I can't get it back. I'm using WP v 4.2.2 and Inblot v 4.5.2

**Comment by mgsisk** [comment]
Apologies for the trouble @CAMorgan. It looks like an upgrade function didn't work quite like it was supposed to. The CSS classes that control how sidebars are displayed changed slightly in v4.5, and they were *supposed* to be updated.

To fix this, you should be able to go into the Customizer and change your layout from whatever it is now to anything else (maybe a two or three column layout; that will preserve your widgets) and back again. You may need to actually save the change first, i.e. change it to "three columns centered", save, then change back.

**Comment by CAMorgan** [comment]
heh, easy enough. All's fixed, thanks!

**Comment by Cryoclaire** [comment]
Hi! I had the same issue after updating Inkblot today - I managed to restore the sidebar in Firefox, however the Chrome version of the website is completely broken - no sidebar, and it displays section names like "comment header" etc. I tried accessing Customizer features through Chrome, but it doesn't help. Anything I could do? WP 4.2.3 and Inkblot 4.5.2

**Comment by foreachthing** [comment]
Yeah, me too! What Cryoclaire wrote.
Just installed (fresh, no update) Inkblot and changed the menu (don't really know if that was the cause) and now the widgets are gone!

EDIT: Even after a really fresh install (remove theme, clean database from inkblot), there is just the standard text from the sidebar.php visible. If I got to the widgets (customizer) it tells me, that there are no widget areas in this theme defined.

Sorry, screenshot is in German: ![inkblot_nowidgets](https://cloud.githubusercontent.com/assets/10420187/11681365/d4bc643a-9e5e-11e5-8416-9d8740e16e27.PNG)

Any ideas?
WP 4.4 and Inkblot 4.5.2
Firefox

**Comment by mgsisk** [comment]
Apologies for not responding sooner @Cryoclaire and @foreachthing. I'll admit, I'm not entirely sure what could be going on. Out of curiosity, is there any difference on the **Appearance > Widgets** page?

**Comment by foreachthing** [comment]
@mgsisk , no, there was no difference! The widgets loaded completely! That's what I didn't understand. Why would the widgets area load (in Appearance > Widgets) but then tell me that there are no areas!?
Funny thing is: after changing the $sidebars array (in sidebars.php) by removing the localization of item [0],  the widgets start working!!! :-o
So, I got it working for me - also made a child-theme, that way I don't have to redo my changes after an update ;-)
If you need more information to debug, let me know.

(see also here: https://wordpress.org/support/topic/widgets-stopped-working-1)
